projects/buttons/web/handlers/instancehandlers.py

```diff
@@ -639,29 +639,55 @@ def post(self, sid=None): # a POST here is a create instance event
                 if name == "failed":
                     raise Exception("Instance start failed.")
 
-                # set up an instance 
-                instance = Instance(
-                    name = name,
-                    status = "PROVISIONING",
-                    user = user_info.key,
-                    stream = stream.key,
-                    size = size,
-                    password = password,
-                    expires = datetime.datetime.now() + datetime.timedelta(0, 604800),
-                    started = datetime.datetime.now()
-                )
-                instance.put()
+                is_alpha = re.compile("^[a-zA-Z]+$")
 
-                slack.slack_message("Instance type %s created for %s!" % (stream.name, user_info.username))
+                # Fusion 5
+                if is_alpha.match(name) and len(name) == 4:
```

Comment:
need to make sure this matches our final agreed upon length

Reply:
this is fine for now